doc/articles/uno-development/guidelines-for-pull-requests.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Guidelines for pull-requests
+---
+
+## Reviewing
+
+Maintainers, contributors and the community can participate in reviewing pull-requests. We require `two approvals` before the pull-request can be merged. Please apply the appropriate labels to the pull-request when reviewing. If the pull-request requires minor touch ups, consider doing them in the GitHub editor rather than asking the initiator of the pull-request to do them for you.
+
+## Merging
+
+We typically don't merge pull-requests by hand, instead we rely on automation and a process of the pull-request initiator adding the `ready-for-merge` label. If the person who initiated the pull-request does not have permission to add a label then the reviewers can add it when the contribution is ready to ship:
```

Comment:
Is there a reason to deprecate manual merges? Automated merges are certainly convenient but I'm not sure why they have to be the 'right way' to do it.

Reply:
I'd like to deprecate manual merges and make them the exception rather than the norm but not enforce this with tooling/configuration so that we have an escape hatch if something is timing sensitive. Mergify will automatically retry merge the contents of master into the pull-request, re-triggering CI and thus ensuring that merging an aged pull-request will not break master.

Reply:
Ok, makes sense.